starters/deno-oak-denodb/src/rest/services/healtcheck_service.ts

```diff
@@ -0,0 +1,39 @@
+import { cache } from '../../cache/mod.ts';
+import { logger } from '../../util/logger.ts';
+import { db } from '../../db/db.ts';
+import { PostgresConnector } from '../../../deps.ts';
+
+export async function isCacheRunning(): Promise<boolean> {
+	let isRunning;
+
+	try {
+		isRunning = await cache.pingRedis() === 'PONG';
+	} catch (e) {
+		isRunning = false;
+		logger.error(e);
+	}
+
+	return isRunning;
```

Comment:
So far, so good.

Have you checked what happens when you stop the redis container?
In node world with the node redis client the healthcheck endpoint was spinning because it was waiting for the request to time out. If the deno redis client works the same, this might be improved with a Promise.race trick.

WDYT? 🤔

Reply:
Yeah, good idea - I added that.